frontend/src/app/api/userApi.ts

```diff
@@ -0,0 +1,76 @@
+export type User = {
+  email: string;
+  firstName: string;
+  lastName: string;
+  role: string;
+  phoneNumber: string;
+  zipCode: number;
+  address: {
+    streetAddress1: string;
+    streetAddress2: string;
+    city: string;
+    state: string;
+  };
+  roleSpecificInfo: {
+    serviceInfo: {
+      dateServiceEnded: Date;
+      branchOfService: string;
+      currentMilitaryStatus: string;
+      gender: string;
+    };
+    interests: string[];
+    assignedPrograms: string[];
+  };
+  assignedVeterans: string[];
```

Comment:
would this field be optional? Since not all users will have assigned veterans e.g. staff or admin